ci/ray_ci/test_windows_container.py

```diff
@@ -42,6 +42,8 @@ def test_get_run_command() -> None:
         "-i",
         "--rm",
         "--env",
+        "BUILDKITE",
+        "--env",
```

Comment:
maybe programmatically generate this `--env` list here based on `_DOCKER_ENV` ?

Reply:
good idea